connectors/services/base.py

```diff
@@ -14,7 +14,29 @@ class ServiceAlreadyRunningError(Exception):
     pass
 
 
-class BaseService:
+_SERVICES = {}
+
+
+def get_services(names, config):
+    return MultiService(*[get_service(name, config) for name in names])
+
+
+def get_service(name, config):
+    return _SERVICES[name](config)
+
+
+class _Registry(type):
```

Comment:
Why make it private? I think this class can be pretty public :)

Reply:
this metaclass is an internal thing for the base class, I don't see any reason to have it public for now, unless we reuse it in another class